invokeai/backend/stable_diffusion/diffusion/custom_atttention.py

```diff
@@ -9,6 +10,12 @@
 from invokeai.backend.stable_diffusion.diffusion.regional_prompt_data import RegionalPromptData
 
 
+@dataclass
+class IPAdapterAttentionWeights:
+    ip_adapter_weights: Optional[IPAttentionProcessorWeights]
+    skip: bool
```

Comment:
`ip_adapter_weights` should not be optional

Reply:
Fixed that. This was a trickle down effect from realizing that the list being passed to the main class can be optional. So I thought we can catch this elsewhere but you're right. This shouldn't be optional in the first place. But the other issue still stands. Resolved this.